### PR TITLE
Features/management

### PR DESCRIPTION
**Pull request by gupele**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Thu Jan 29 14:31:06 IST 2015
+#Thu Aug 06 14:30:21 IDT 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-all.zip
```
Comment (amnonshmsft) [inline gradle/wrapper/gradle-wrapper.properties]: Why did you change it?

File: management/build.gradle [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+archivesBaseName = 'applicationinsights-management'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    // TODO: fix this! required?
+    compile files('C:\\Sources\\azure-activedirectory-interactive-auth-library-for-java.jar')
```
Comment (amnonshmsft) [inline management/build.gradle]: Please fix this path before pushing.
Also, please make sure that build will not break on systems that do not have this file available.

File: management/src/test/java/Program.java [hunk 3]
```diff
@@ -0,0 +1,158 @@
+import com.microsoft.applicationinsights.management.authentication.Authenticator;
```
Comment (amnonshmsft) [inline management/src/test/java/Program.java]: License please

File: management/src/test/java/com/microsoft/applicationinsights/management/rest/operations/GetResourcesOperationTests.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
+package com.microsoft.applicationinsights.management.rest.operations;
```
Comment (amnonshmsft) [inline management/src/test/java/com/microsoft/applicationinsights/management/rest/operations/GetResourcesOperationTests.java]: License please

File: management/src/test/java/com/microsoft/applicationinsights/management/rest/operations/GetSubscriptionsOperationTests.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
+package com.microsoft.applicationinsights.management.rest.operations;
```
Comment (amnonshmsft) [inline management/src/test/java/com/microsoft/applicationinsights/management/rest/operations/GetSubscriptionsOperationTests.java]: License please

File: settings.gradle [hunk 6]
```diff
@@ -1,25 +1,3 @@
...
- * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
- * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
- * PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
- * FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
- * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
- * DEALINGS IN THE SOFTWARE.
- */
-
-
 include 'agent'
```
Comment (amnonshmsft) [inline settings.gradle]: Why was the license statement removed?

File: settings.gradle [hunk 7]
```diff
@@ -30,6 +8,7 @@ include 'distributions'
 include 'samples'
 include 'test:performance'
 include 'test:webapps:bookstore-spring'
+include 'management'
```
Comment (amnonshmsft) [inline settings.gradle]: As this module requires an external dependency, if this module is not required for all builds please condition is as done with collectd below

**Comment by littleaj** [comment]
This pull request is very old and is being pruned. The branch still exists if someone thinks this should be reopened.
